vendor/views/vendor_admin.py

```diff
@@ -358,3 +359,35 @@ def post(self, request, *args, **kwargs):
 
         messages.info(request, _("Subscription Renewed"))
         return redirect(request.META.get('HTTP_REFERER', self.success_url))
+
+
+class PaymentWithNoReceiptListView(LoginRequiredMixin, ListView):
+    template_name = "vendor/manage/payment_list.html"
+    model = Payment
+
+    def get_queryset(self):
+        if hasattr(self.request, 'site'):
```

Comment:
do you need this in here if you're setting the site from the request in line 371?

Reply:
yeah removed that unnecessary check if the function already does it